CHANGES.md

```diff
@@ -2,6 +2,15 @@
 
 ## __NEXT__
 
+### Features
+
+- Support newer versions of jsonschema (≥4.18.0). [#1691] (@victorlin)
```

Comment:
is it "support" or is it "require"?

Reply:
Good point. Technically this is a major change then, similar to

https://github.com/nextstrain/augur/blob/e90383bf99e4d6758a40c9140998be63f186d6ef/CHANGES.md#L389-L394

Reply:
Updated to

https://github.com/nextstrain/augur/blob/67ada119957704fe8629e41b38b143edd730b10e/CHANGES.md#L5-L7